hw/ip/spi_device/doc/_index.md

```diff
@@ -46,6 +46,8 @@ title: "SPI Device HWIP Technical Specification"
 - 1 TPM command (8b) and 1 address (24bit) buffer
 - HW controlled wait state
 - Shared SPI with other SPI Device functionalities. Unique CS# for the TPM
+  - Flash or Passthrough mode can be active with TPM mode.
+    Generic and TPM modes are mutually exclusive.
```

Comment:
@eunchan does this mean the plan to merge tpm function into the 2p sram won't work? or is it a matter of carving out some space for tpm specifically?

Reply:
As TPM mode cannot be enabled with Generic mode, we can move TPM RX/TX FIFOs into DPSRAM